src/libraries/System.Text.Json/gen/JsonSourceGenerator.Parser.cs

```diff
@@ -103,10 +105,6 @@ private sealed class Parser
             private readonly Type _intPtrType;
             private readonly Type _uIntPtrType;
 
-            // Unsupported types that may not resolve
```

Comment:
Shouldn't this remain unsupported for < .NET 6? Where is this tested?

Reply:
We discussed this during API review, and came to the conclusion that the source generator should bind to `DateOnly` and `TimeOnly` so long as a type matching the full name has been found. So any well-typed POCO containing `System.DateOnly` or `System.TimeOnly` members would trigger source generation regardless of whether the runtime contains the designated types or its converter definitions.